nethserver-tancredi.spec

```
@@ -27,6 +28,7 @@ mv tancredi/src root/usr/share/tancredi/
 mv tancredi/vendor root/usr/share/tancredi/
 mv tancredi/data/templates root/usr/share/tancredi/data/
 mv tancredi/data/patterns.d root/usr/share/tancredi/data/
+cp %{S:2} root/usr/share/tancredi/src/Entity/
```

The `Source2` tag is not necessary, as the file is already bundled within Source0 .tar.gz

```suggestion
cp NethVoiceAuth.php root/usr/share/tancredi/src/Entity/
```